server/src/middlewares/measurements.spec.js

```diff
@@ -0,0 +1,66 @@
+const { measurement } = require('./measurements');
+const data = require('../data');
+
+describe('measurements', () => {
+
+  describe('measurement ', () => {
+    const measurementObj = { measurableId: 'measurableId1' };
+    beforeAll(() => {
+      jest.spyOn(data.measurementData, 'getOne').mockReturnValue(
+        Promise.resolve(measurementObj),
+      );
+    });
+
+    it('A', () => {
```

Comment:
Can you give a description? "A" doesn't mean anything to me.

Reply:
Fixed.